subscription-engine/kafka-ws/app/Redis/RedisManager.js

```diff
@@ -42,6 +44,18 @@ class RedisManager {
     return Object.seal(this);
   }
 
+  redisManagerStrategy(options) {
+    // reconnect after
+    logger.debug(`Retry strategy options ${util.inspect(options, false, 5, true)}`);
+
+    if (options.attempt === this.config['max.attemps']) {
+      StateManager.shutdown();
+    }
+
+    // return timeout to reconnect after
+    return this.config['strategy.connect.timeout'];
```

Comment:
It would be better to use an exponential backoff here. Also, the name 'strategy.connect.timeout' can lead to a misunderstanding. A better name would be reconnect.after.